### PR TITLE
new alcatel_sros tmpl, 'show service sdp'

### PR DESCRIPTION
**Pull request by h4ndzdatm0ld**
- New Template Pull Request

alcatel_sros_show_service_sdp,
alcatel_sros,
'show service sdp'

Additional template submission

### CODE REVIEW

File: templates/alcatel_sros_show_service_sdp.textfsm [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+SDP
+  ^${SDP_ID}\s+${ADM_MTU}\s+${OPR_MTU}\s+${FAR_END}\s+${ADM}\s+${OPR}\s+${DEL}\s+${LSP}\s+${SIG} -> Record
+  ^${SDP_ID}\s+${ADM_MTU}\s+${OPR_MTU}\s+${ADM}\s+${OPR}\s+${DEL}\s+${LSP}\s+${SIG} -> Record
+  ^-+
+  ^Number
+  ^Legend
+  ^\s+I\s=
+  ^\s*$$
+  ^=+
+  ^. -> Error
```
Comment (jmcgill298) [inline templates/alcatel_sros_show_service_sdp.textfsm:28]:
```suggestion
  ^. -> Error

Done
```

File: templates/alcatel_sros_show_service_sdp.textfsm [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  ^\s*$$
+  ^. -> Error
+
+SDP
+  ^${SDP_ID}\s+${ADM_MTU}\s+${OPR_MTU}\s+${FAR_END}\s+${ADM}\s+${OPR}\s+${DEL}\s+${LSP}\s+${SIG} -> Record
+  ^${SDP_ID}\s+${ADM_MTU}\s+${OPR_MTU}\s+${ADM}\s+${OPR}\s+${DEL}\s+${LSP}\s+${SIG} -> Record
+  ^-+
+  ^Number
+  ^Legend
+  ^\s+I\s=
```
Comment (jmcgill298) [inline templates/alcatel_sros_show_service_sdp.textfsm:25]:
```suggestion
  ^-+ -> Done
```

**Review by jmcgill298 — CHANGES_REQUESTED** [review]
Thanks @h4ndzdatm0ld

Matching `Legends` can be difficult, I believe my suggestions should work and avoid that section

**Comment by h4ndzdatm0ld** [comment]
Thanks @jmcgill298  - Moving to the Done state and dealing with the legends is great. I will keep this in mind for the next one. I committed the changes, tested and ran against a device - looks good!

**Review by jmcgill298 — APPROVED** [review]

**Pull request merged**